core/src/components/radio/radio.ios.scss

```diff
@@ -70,13 +72,28 @@
 // iOS Radio Within An Item
 // -----------------------------------------
 
-:host(.in-item) {
+:host(.in-item.legacy-radio) {
   @include margin($radio-ios-item-end-margin-top, $radio-ios-item-end-margin-end, $radio-ios-item-end-margin-bottom, $radio-ios-item-end-margin-start);
 
   display: block;
   position: static;
 }
 
-:host(.in-item[slot="start"]) {
+:host(.in-item.legacy-radio[slot="start"]) {
   @include margin($radio-ios-item-start-margin-top, $radio-ios-item-start-margin-end, $radio-ios-item-start-margin-bottom, $radio-ios-item-start-margin-start);
 }
+
+// Radio Wrapper
+// ----------------------------------------------------------------
+
+.radio-wrapper {
+  min-height: 42px;
```

Comment:
Can we move this to the `:host` and have .radio-wrapper inherit it? This was developers can customize it by targeting `ion-radio` in their CSS.

Note: I fixed this in input as well: https://github.com/ionic-team/ionic-framework/commit/0cd6e918e83ff4c2cdc41e1e9da05fefe6ca5c84